.github/workflows/main.yml

```diff
@@ -22,4 +25,4 @@ jobs:
           - windows-latest
         node:
           - lts/erbium
-          - node
```

Comment:
Only the change of this line should be enough? You should be able to skip all others. See https://github.com/unifiedjs/unified/commit/6f6a5c207e2d69cd99da99561c45989ac84dc411

Reply:
Thanks! I think the https://github.com/dcodeIO/setup-node-nvm step failed to install Node.js in that commit's [:point_up:](https://github.com/unifiedjs/unified/commit/6f6a5c207e2d69cd99da99561c45989ac84dc411) `lts/gallium on windows-latest` check: https://github.com/unifiedjs/unified/runs/4991302576?check_suite_focus=true#step:3:1

Consequently the check passed, but version 16.13.2 was used?
- https://github.com/unifiedjs/unified/runs/4991302576?check_suite_focus=true#step:1:9
- https://github.com/actions/virtual-environments/blob/win19/20220123.1/images/win/Windows2019-Readme.md#language-and-runtime

I gave it a try today and confirmed version 16.14.0 is affected ([same as 17.4.0](https://github.com/unifiedjs/unified/runs/4991281393?check_suite_focus=true#step:3:16)):
- https://github.com/remarkjs/remark-lint/runs/5259399904?check_suite_focus=true#step:3:16
- https://github.com/remarkjs/remark-lint/runs/5259399904?check_suite_focus=true#step:4:4

Avoiding [older nvm-windows](https://github.com/dcodeIO/setup-node-nvm/blob/3bc09a3f6f58358a453eccc31a88b0c661cba9f9/install.ps1#L6), by replacing https://github.com/dcodeIO/setup-node-nvm with https://github.com/actions/setup-node, solves that problem.